docs/terminology.rst

```diff
@@ -2,6 +2,102 @@
 Terminology
 ===========
 
+Naming objects in and around the DDM
```

Comment:
This documentation *could* be less verbose, using for example more lists of keywords instead of full sentences. But I already like it a lot the way it is, so you can also just leave it as is.

Reply:
I disagree, I think making it more terse would make it harder to read. The vocabulary for ODRL has a few nice examples of terse definitions that are difficult to comprehend because there's no context, e.g. https://www.w3.org/TR/odrl-vocab/#term-informingParty. And the parts of it that are understandable are at least as verbose, e.g. https://www.w3.org/TR/odrl-model/#asset.